client/cli/src/commands/import_blocks_cmd.rs

```diff
@@ -39,6 +39,10 @@ pub struct ImportBlocksCmd {
 	#[structopt(long = "default-heap-pages", value_name = "COUNT")]
 	pub default_heap_pages: Option<u32>,
 
+	/// Try importing blocks from binary format rather than JSON.
+	#[structopt(long = "binary", value_name = "BOOL", parse(try_from_str), default_value("false"))]
```

Comment:
```suggestion
	#[structopt(long)]
```

won't just this work ^^^ ?

Reply:
YES :grin:

Reply:
Haha so I copied this from the `export-blocks` command which ^^ gonna modify both of these bad boys! thanks :)